vuln-remediation.md

```diff
@@ -0,0 +1,105 @@
+      > 
+      > [include if needed] Based on the severity of the vulnerability, the details are particularly sensitive and need to be protected through secure communications. Please send me your public PGP key through which you can receive secure email; if you are unable to send and receive secure email, please let me know and we will establish another secure channel.
+      > 
+      > [Attach a PGP key or a Key ID]
+      >
+      > [Sign the email with the PGP key]
+      >
+
+   1. **Coordinator registers a CVE.** Seek help from the [ROS 2 security working group](https://github.com/ros-security/community#communication-channels) if necessary to reserve the CVE. The reserved CVE should not include any detailed information; this will be added after disclosure. All subsequent communications should include the CVE numnber for traceability.
+
+1. **Remediation**
+
+   1. **Maintainer fixes the vulnerability.** Work should be done locally as much as possible; when the fix is pushed to github the fix becomes visible to the public even though the patch has not been released to users. The final pull request for the fix should include a reference to the CVE in comments for traceability.
+
+   1. **Coordinator tracks to completion.**  This likely will mean periodic update requests to the Maintainer until the Maintainer publishes a fix. Seek assistance from the Security WG if needed.
+
+   1. **Coordinator plans for vulnerability disclosure.** For high risk vulnerabilities or fixes with significant impact, create a communications plan for patch release and full disclosure. The plan should take input from the Maintainer and the Reporter.
+
+1. **Disclosure**
+
+   1. **Maintainer conducts patch pre-notification.** When the fix is ready for release, the Maintainer, at his or her discretion, will notify select individuals that the fix is ready. This notification should be specifically to quickly remediate any build issues caused by the patch, to merge the pull request, monitor as the patch is bloomed, and finally confirm the fix is sync'd into the ROS distro.
+
+   1. **Coordinator notifies the Reporter that the patch has been released.** At this point the patch is available as an update to ROS users.
+
+   1. **Disclose the patch to a broader audience as desired.** Follow the vulnerabilty disclosure plan if one exists. Consider notifying the ROS community of the patch availability; for example, with a general post to [ROS discourse](https://discourse.ros.org/).
+
+   1. **Coordinator updates CVE details.** Include a link to the patch and final CVSS details. 
+
+
+## Additional Guidance
+
+### Identifying the Maintainer
+
+If the Maintainer of the vulnerable package is not well known, check for a <maintainer> tag in the package's `package.xml` file. Also review recent activity for the package, and contact a recent pull request approver(s) to find a responsible person.
```

Comment:
```suggestion
If the Maintainer of the vulnerable package is not well known, check for a <maintainer> tag in the package's `package.xml` file. Also review recent activity for the package, and contact a recent pull request merger(s) to find a responsible person.
```

People reviewing / approving Pull Requests may not be the same people as the ones merging and releasing code updates

Reply:
Updated in 44179b32ecadc54d74ed05f01ba81d9b3b58c2c9